megamek/src/megamek/common/Entity.java

```diff
@@ -7256,8 +7256,18 @@ public PilotingRollData checkRecklessMove(MoveStep step,
 
         // this only applies in fog, night conditions, or if a hex along the
         // move path has ice
+        int minTemp = -30;
```

Comment:
Minimum temp for what? The name is ambiguous. Also, as its only used in one place, I don't see the utility in hoisting the constant into a local.

Small nit, the value of `-30` does not represent the _minimum_ temperature below which black ice does not form, but rather the maximum temperature above which black ice does not form. `maxTempForBlackIceFormation`?